src/InstallDir/InstallDirDialog.jsx

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2015 - 2020, Nordic Semiconductor ASA
+/* Copyright (c) 2015 - 2018, Nordic Semiconductor ASA
```

Comment:
I did not change this, git just wrongly believes I created `src/InstallDir/InstallDirDialog.jsx` from `src/Settings/settingsActions.js` and so compares them. Generally I think it would be a lot better for us to get away with these numbers. I have seen others do that.

E.g. in https://github.com/facebook/react/commit/bef45b0b1a98ea9b472ba664d955a039cf2f8068 Facebook switched from lines like <br />`Copyright (c) 2013-2015, Facebook, Inc.` to <br />`Copyright (c) 2013-present, Facebook, Inc.`. 

And then in https://github.com/facebook/react/commit/b87aabdfe1b7461e7331abb3601d9e6bb27544bc then went further to just <br />`Copyright (c) Facebook, Inc. and its affiliates.`.

Alternatives would be to have scripts to automate this. Either update all files at the beginning of the year or update a file when needed on commit. But the current practice of putting the years there by hand just does not work.

Reply:
We're using a company wide template for the licenses, so if we want to change we should modify the template.
https://projecttools.nordicsemi.no/confluence/display/QPDA/4-Clause+Nordic+License+text